oak-lucene/src/main/java/org/apache/jackrabbit/oak/plugins/index/lucene/hybrid/NRTIndex.java

```diff
@@ -94,6 +96,8 @@ public class NRTIndex implements Closeable {
     private List<LuceneIndexReader> readers;
     private final List<IndexReader> openedReaders;
     private final boolean assertAllReadersClosed;
+    private final StatsProviderUtil statsProviderUtil;
+    private Map<String, String> labels;
```

Comment:
this could be final too
```suggestion
    private final Map<String, String> labels;
```

Reply:
done